components/ButtonBar/src/private/MDCAppBarButtonBarBuilder.m

```diff
@@ -49,10 +50,10 @@
 @interface MDCButtonBarSandbagView : UIView
 @end
 
-@interface MDCButtonBarButton : MDCFlatButton
+@interface MDCButtonBarButton (MDCAppBarButtonBarBuilder)
 
-// Content padding for the button.
-@property(nonatomic) UIEdgeInsets contentPadding;
+//// Content padding for the button.
+//@property(nonatomic) UIEdgeInsets contentPadding;
```

Comment:
Should these commented lines be removed?

Reply:
Done.